server/models/Group.js

```diff
@@ -394,15 +398,24 @@ module.exports = function(Sequelize, DataTypes) {
           }
         })
         .then(userGroup => Promise.resolve(!!userGroup));
+      },
+
+      getSuperCollectiveData() {
```

Comment:
(to be clear, that shouldn't prevent us from merging this PR, it's definitely good enough for now)